package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "photoviewer",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "analyze": "cross-env ANALYZE=true next build",
+    "analyze:server": "cross-env BUNDLE_ANALYZE=server next build",
+    "analyze:browser": "cross-env BUNDLE_ANALYZE=browser next build",
+    "analyse:base": "madge src/ --extensions ts,tsx",
+    "analyse:unused": "node scripts/detect-unused.js",
+    "analyse:circular": "yarn analyse:base --circular",
+    "analyse:orphans": "yarn analyse:base --orphans",
+    "format": "prettier --write \"**/*.+(ts|tsx|css|md)\"",
+    "format:check": "prettier --check .",
+    "lint": "eslint --ext .ts,.tsx",
+    "lint:fix": "eslint --ext .ts,.tsx --fix .",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@datapunt/matomo-tracker-react": "^0.5.1",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
+    "@ethersphere/bee-js": "^3.3.4",
+    "@types/react-blockies": "^1.4.1",
+    "axios": "^0.21.1",
+    "ethers": "^5.6.5",
+    "file-saver": "^2.0.5",
+    "moment": "^2.29.1",
+    "next": "^11.1.3-canary.7",
+    "pretty-bytes": "^6.0.0",
+    "react": "^17.0.2",
+    "react-blockies": "^1.4.1",
+    "react-dom": "17.0.2",
+    "react-hook-form": "^7.22.0",
+    "swarm-nft": "https://github.com/igar1991/SwarmNFT.git"
```

Comment:
should we move this to fairDataSociety  too?

Reply:
I think, No.